docs/_include/mooring.rst

```diff
@@ -135,8 +148,9 @@ Applications <https://github.com/WEC-Sim/WEC-Sim_Applications>`_ repository.
 
 * **MoorDyn Input File**: A mooring folder that includes a moorDyn input file 
   (``lines.txt``) is created. The moorDyn input file (``lines.txt``) is shown 
-  in the figure below. More details on how to setup the MooDyn input file are 
-  described in the MoorDyn User Guide :cite:`Hall2015MoorDynGuide`.
+  in the figure below. More details on how to set up the MooDyn input file are 
```

Comment:
@jtgrasb Does the current configuration always require a coupled body on the MD side? If so I can make that change in the MD documentation, or we could put it on this end

Reply:
Technically, I believe another body could be added to lines.txt and simulated in MoorDyn. It would be a weird way to do it but possible. As for the WEC-Sim body, it should always be specified as Coupled, so I have added this to the docs here.